packages/core/src/components/CopyTextButton/CopyTextButton.tsx

```diff
@@ -44,23 +44,36 @@ const useStyles = makeStyles(theme => ({
  * Example:
  *    <CopyTextButton text="My text that I want to be copied to the clipboard" />
  */
-const CopyTextButton = ({
-  text,
-  tooltipDelay = 1000,
-  tooltipText = 'Text copied to clipboard',
-}) => {
-  const classes = useStyles();
+type Props = {
+  text: string;
+  tooltipDelay?: number;
+  tooltipText?: string;
+};
+
+const defaultProps = {
+  tooltipDelay: 1000,
+  tooltipText: 'Text copied to clipboard',
+};
+
+const CopyTextButton: FC<Props> = props => {
+  const { text, tooltipDelay, tooltipText } = {
+    ...defaultProps,
+    ...props,
+  };
+  const classes = useStyles(props);
   const errorApi = useApi(errorApiRef);
-  const inputRef = useRef();
+  const inputRef = useRef<HTMLInputElement>(null);
   const [open, setOpen] = useState(false);
 
   const handleCopyClick = e => {
     e.stopPropagation();
     setOpen(true);
 
     try {
-      inputRef.current.select();
-      document.execCommand('copy');
+      if (inputRef.current) {
```

Comment:
👍 , but this seems unlikely to be false x)

Reply:
That's true, but the TypeScript check for null/undefined values requires us to do this. Or else, it will keep complaining that "inputRef.current can be null".

Reply:
You can use `inputRef.current!.select()` is you're sure it's defined to keep things shorter, but I prefer the `if` here anyway

Reply:
> inputRef.current!.select()

Oh yeah, that is short. I found this syntax on a Stackoverflow answer a while ago, but immediately forgot what it does. :P
`if` is indeed simple, thanks!